### PR TITLE
Several Translation issues

### LINKED ISSUE #19166: "Notehead scheme" in staff properties different from "Notehead system" in Properties panel
### Issue type

UI bug

### Bug description

Just flagging up a UI labelling issue.

In Staff Properties > Advanced style properties there is a **Notehead scheme** dropdown. There is a similar dropdown in the Properties panel called **Notehead system**.

The two dropdowns work slightly differently (as explained [here](https://musescore.org/en/handbook/4/noteheads#change-schemes)), but, AFAICS, they should be called the same thing.

### MuseScore Version

MuseScore version (64-bit): 4.1.1-232071203, revision: github-musescore-musescore-e4d1ddf

### Regression

N/A

### Operating system

OS: Linux Mint 20.1, Arch.: x86_64,

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
* Differentiating the meanings of "Minor" and "Major"
  Resolves: #19153
* "notes" to "keys", as this is about the piano keyboard
* Sentence case vs. Title Case in transpose dialog
* Sentence case vs. Title Case in breaks dialog
* Sentence case vs. Title Case in editpitch dialog
* Sentence case vs. Title Case in editstaff dialog
* Sentence case vs. Title Case in editstafftype dialog
* Sentence case vs. Title Case in editstringdata dialog
* Sentence case vs. Title Case in keyedit dialog
* Sentence case vs. Title Case in masterpalette dialog
* Sentence case vs. Title Case in measureproperties dialog
* Sentence case vs. Title Case in pagesettings dialog
* Sentence case vs. Title Case in realizeharmony dialog
* Sentence case vs. Title Case in selectnote dialog
* Sentence case vs. Title Case in specialcharacters dialog
* Sentence case vs. Title Case in stafftextproperties dialog
* Sentence case vs. Title Case in tuplet dialog
* Sentence case vs. Title Case in voicingselect dialog
* Sentence case vs. Title Case in voicingselect dialog
* Sentence case vs. Title Case in editstyle dialog
* Sentence case vs. Title Case in editdrumset dialog
* Plural for palettes
* English plurals rather than Italian ones
* Use winged double quotes (as per discussion in #17748)
   Not sure whether the single quotes should also get changed to winged ones, there are some 140 strings affected, the vast majority of them in the editstyle dialog, which is slated to get converted to QML
* Better naming, "Leading" to "Leading space"
* "Notehead scheme" in staff properties different from "Notehead system" in Properties panel
   Resolves #19166

### CODE REVIEW

**Review by shoogle — CHANGES_REQUESTED** [review]
I see this in different files:

```C++
//: mode of a key signature, not an interval
qsTrc("inspector", "Major")
```
```C++
//: Interval, not the mode of a key signature
qsTrc("inspector", "Major")
```

However, **translation comments are not considered during translation lookup**, so both strings will use the same translation for "Major".

To get a different translation, you must either change the **context** string:
```C++
qsTrc("inspector/keysignatures", "Major")
```
```C++
qsTrc("inspector/ornaments", "Major")
```
Or pass in a **disambiguation** string as an additional argument to `qsTrc()`:
```C++
qsTrc("inspector", "Major", "key signature mode")
```
```C++
qsTrc("inspector", "Major", "interval quality")
```

I'll let @cbjeukendrup say which he would prefer in this case.

The context and disambiguation are displayed on Transifex, but they are buried quite far down the page, so it is helpful to provide the translation comments as well.

File: src/notation/view/widgets/transposedialog.ui
Comment (shoogle) [inline src/notation/view/widgets/transposedialog.ui:337]: Typo: octave

**Comment by Jojo-Schmitz** [comment]
For now I've picked the disabiguation.

File: src/inspector/view/qml/MuseScore/Inspector/notation/sectionbreaks/SectionBreakSettings.qml
Comment (shoogle) [inline src/inspector/view/qml/MuseScore/Inspector/notation/sectionbreaks/SectionBreakSettings.qml:67]: Please don't add hard-coded newlines to UI strings.

1. The result doesn't look good.
    image
2. It's not even necessary. Qt will wrap the string automatically if it's too long. (See [`wrapMode: Text.Wrap`](https://doc.qt.io/qt-6/qml-qtquick-text.html#wrapMode-prop))
    image
    I had to write 'looooonnng' because the normal string actually fits at the default text size, so there is no need to wrap it at all!

Also, I notice that you added a line break to this string in the English US and English GB translations. Please don't use the English translations to fix UI bugs (not that there is a bug in this case). If you think there's a UI bug, open an issue for it.

Please remove the line break here, and also remove it from the translations for this string and any others like it.

File: src/inspector/view/qml/MuseScore/Inspector/notation/sectionbreaks/SectionBreakSettings.qml
Comment (shoogle) [inline src/inspector/view/qml/MuseScore/Inspector/notation/sectionbreaks/SectionBreakSettings.qml:76]: Not sure we really need these warnings. Qt will wrap the strings if they are too long. If they don't wrap then it's a bug. It would mean we need to set the [`wrapMode` property](https://doc.qt.io/qt-6/qml-qtquick-text.html#wrapMode-prop) somewhere.

File: src/project/qml/MuseScore/Project/internal/NewScore/KeySignatureSettings.qml
Comment (shoogle) [inline src/project/qml/MuseScore/Project/internal/NewScore/KeySignatureSettings.qml:101]: Might as well add the disambiguation here too. We should probably get in the habit of doing this everywhere we use "Major" and "Minor".

File: src/project/qml/MuseScore/Project/internal/NewScore/KeySignatureSettings.qml
Comment (shoogle) [inline src/project/qml/MuseScore/Project/internal/NewScore/KeySignatureSettings.qml:110]: Same here.

**Review by shoogle — CHANGES_REQUESTED** [review]

File: src/appshell/view/internal/splashscreen/newinstanceloadingscreenview.cpp
Comment (Jojo-Schmitz) [reply src/appshell/view/internal/splashscreen/newinstanceloadingscreenview.cpp:42]: Oops, to subtle a difference for my aging eyes ;-)

File: src/inspector/view/qml/MuseScore/Inspector/notation/sectionbreaks/SectionBreakSettings.qml
Comment (Jojo-Schmitz) [reply src/inspector/view/qml/MuseScore/Inspector/notation/sectionbreaks/SectionBreakSettings.qml:67]: Ah, that must be new with Mu4 then and a leftover for a fix needed in Mu3.

File: src/inspector/view/qml/MuseScore/Inspector/notation/sectionbreaks/SectionBreakSettings.qml
Comment (Jojo-Schmitz) [reply src/inspector/view/qml/MuseScore/Inspector/notation/sectionbreaks/SectionBreakSettings.qml:76]: yep

File: src/project/qml/MuseScore/Project/internal/NewScore/KeySignatureSettings.qml
Comment (Jojo-Schmitz) [reply src/project/qml/MuseScore/Project/internal/NewScore/KeySignatureSettings.qml:101]: OK, there are only those 2 time 3, as far as I can tell

File: src/project/qml/MuseScore/Project/internal/NewScore/KeySignatureSettings.qml
Comment (Jojo-Schmitz) [reply src/project/qml/MuseScore/Project/internal/NewScore/KeySignatureSettings.qml:110]: sure

**Review by shoogle — APPROVED** [review]
Looks good to me, but I'd like to get sign-off from a designer before we merge.

**Comment by cbjeukendrup** [comment]
I still doubt very much about whether we should do that sentence case thing with those old dialogs. It gives translators a lot of useless extra work, and the benefits are very, very small, considering that these dialogs are to be completely overhauled anyway. (And I'm inclined to say that these dialogs are so ugly anyway that some capitalisation inconsistencies that only 0.03% of people will ever notice are not going to make much of a difference.)

**Comment by Jojo-Schmitz** [comment]
It isn't that many strings, some 125

**Comment by shoogle** [comment]
I think the [Translation Memory](https://help.transifex.com/en/articles/6224636-introduction-to-translation-memory) should recognise these strings as being similar to the previous ones and provide the old translation as a suggestion, possibly even fill it in [automatically](https://help.transifex.com/en/articles/6224817-setting-up-translation-memory-fill-up), so all translators have to do is review.

File: src/engraving/libmscore/jump.cpp
Comment (shoogle) [inline src/engraving/libmscore/jump.cpp:62]: @oktophonie, what do you think of these?

File: src/notation/view/selectionfiltermodel.cpp
Comment (shoogle) [inline src/notation/view/selectionfiltermodel.cpp:199]: @oktophonie

File: src/palette/internal/palettecreator.cpp
Comment (shoogle) [inline src/palette/internal/palettecreator.cpp:219]: We don't want this change.

**Review by shoogle — CHANGES_REQUESTED** [review]
Please stop adding new things to the PR after it has been reviewed. It's ok to make improvements to approved functionality (e.g. if you spot another straight quote than needs to be replaced with a curly quote) but introducing an entirely new kind of change just delays merging further while we go through another round of approvals from all the different stakeholders (i.e. developers, designers, and the engraving team).

Ideally you should open separate PRs for each kind of change:

1. Disambiguate intervals and keys.
2. Replace straight quotes with curly quotes.
3. Use sentence case.
4. Replace "double coda" with "codetta" (this occurs in lots of places).
5. Other string changes (which only occur in one or two places each).

That way each PR only has to be approved by relevant stakeholders rather than by everybody, so changes can get merged faster.

**Comment by shoogle** [comment]
Also, it's not cool to bury one or two high-impact changes (like renaming a palette) in with a plethora of low-impact changes (like replacing straight quotes with curly quotes everywhere). That's exactly the kind of thing that might get merged accidently and then we'd just end up reverting it later on, only we couldn't (easily) revert it because it's combined with other changes, so that means more work... you get the idea!

File: src/palette/internal/palettecreator.cpp
Comment (Jojo-Schmitz) [reply src/palette/internal/palettecreator.cpp:219]: Why not? That palette __does__ contain both

**Comment by Jojo-Schmitz** [comment]
This is why this PR comes with a bunch of commits, you can easily review each and take the entire PR or just cherry-pick some commits and also can easily revert a commit, should it cause trouble

I could squash all the Title Case vs. Sentence case commit into just one, I was using many because it was quite a big task.

**Comment by shoogle** [comment]
Yes, please squash the related commits, and mark the PR as 'draft' if you plan to make further changes.

Multiple commits are useful, but GitHub's 'Viewed' feature (to hide files that you've seen before) only works when you view the full diff, not the per-commit diffs, so there are advantages to using separate PRs.

But the main problem is that you keep adding unrelated changed to the PR after it has been approved. This requires another approval, often by different people, whereas if you only added related changes (like fixing another instance of title case) then we could still merge it based on the original approval.

File: src/palette/internal/palettecreator.cpp
Comment (shoogle) [reply src/palette/internal/palettecreator.cpp:219]: We consider hairpins to be a form of dynamic.

Comment (Jojo-Schmitz) [reply src/palette/internal/palettecreator.cpp:219]: I don't

**Comment by Jojo-Schmitz** [comment]
Done. No further changes planned, so not coverted to draft

File: src/palette/internal/palettecreator.cpp
Comment (oktophonie) [reply src/palette/internal/palettecreator.cpp:218]: That's a pity.

File: src/notation/view/selectionfiltermodel.cpp
Comment (oktophonie) [reply src/notation/view/selectionfiltermodel.cpp:199]: I seem to recall this being discussed in the past, and considered on the basis that we use e.g. "arpeggios" and "tremolos", so it's more consistent to stick to using English plurals in English. I'm fine with it.

File: src/palette/internal/palettecreator.cpp
Comment (Jojo-Schmitz) [reply src/palette/internal/palettecreator.cpp:218]: Oh well. Doesn't translate nicely into German anyway.

File: src/engraving/libmscore/jump.cpp
Comment (oktophonie) [reply src/engraving/libmscore/jump.cpp:62]: I'm not familiar enough with the arcane mysteries of either double codas nor with Italian to say. The language mixture is certainly a bit odd, but whether these changes are 'correct' I don't know.

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/jump.cpp:62]: I'm not sure either, only thing I am sure about is that the Double Coda was called Codetta in Mu1.
And that there is neither Codetta nor Double Coda as a separate/single qlyph in SMuFL

File: src/notation/view/selectionfiltermodel.cpp
Comment (Jojo-Schmitz) [reply src/notation/view/selectionfiltermodel.cpp:199]: I guess that marks this conversation resolved

File: src/palette/internal/palettecreator.cpp
Comment (Jojo-Schmitz) [reply src/palette/internal/palettecreator.cpp]: It is gone now.

File: src/engraving/libmscore/jump.cpp
Comment (Jojo-Schmitz) [reply src/engraving/libmscore/jump.cpp:62]: We could also change "Double" to "Doppio", to avoid that language mix.
I don't really care either way.

Comment (shoogle) [reply src/engraving/libmscore/jump.cpp:62]: @mike-spa, what do you think about 'Double Coda', 'Doppio Coda' and 'Codetta'?

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/jump.cpp:62]: Ah, yes, we do have an Italian in the team ;-)

**Comment by Jojo-Schmitz** [comment]
No change, just a rebase

**Comment by Jojo-Schmitz** [comment]
AKAICT I did the changes that were requested, or didn't I?

**Comment by shoogle** [comment]
You've done what was requested so far, but @mike-spa is away for a couple of weeks so you'll have to drop the "Codetta" changes if you want to see the other changes merged before he returns.

**Comment by Jojo-Schmitz** [comment]
Done and separated out into another PR, #19294

**Review by shoogle — APPROVED** [review]

**Pull request merged**
